lib/checkPullRequestLabels.js

```diff
@@ -1,3 +1,6 @@
+const CRITICAL_LABEL_NAME = 'critical';
+const { releaseCoordinators: RELEASE_COORDINATORS, SERVER_JOBS_ADMIN } = require('../userWhitelist.json');
```

Comment:
Is this line too long, if yes please break.

Reply:
Done.